icone_rtdh_standard.json

```diff
@@ -0,0 +1,88 @@
+[
```

Comment:
Should this go into an examples directory?

Reply:
I don't see it used or mentioned anywhere.

Reply:
removed